docs/source/get_started/access_info/access_info_qss.ipynb

```diff
@@ -0,0 +1,307 @@
+{
```

Comment:
<p>Minor curiosity: do we have a standard for the capitalization of #comments? Some notebook comments begin with capital letters while others begin with lowercase.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SupertechLabs/client-superstaq/pull/543/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
I am not sure - Maybe @richrines1 or @dowusu-antwi would know?

Reply:
Just saw this; we don't have a standard for comments. (Maybe we should)

Reply:
i've wondered about this too. also whether/when they should end in periods vs. colons vs. no punctuation..

currently we're all over the place, so i usually just copy whatever the previous comment in the file does. the inconsistency definitely bothers me though lol

Reply:
Opened this issue so we can keep track of this after June 30: https://github.com/Infleqtion/client-superstaq/issues/560